integration-tests/tests/cases/mpc.rs

```diff
@@ -18,7 +18,7 @@ const KEY_SHARE_FILE: &str = "tmp/key_shares.json";
 const TRIPLES_FILE: &str = "tmp/triples.json";
 const PRESIGNATURES_FILE: &str = "tmp/presignatures.json";
 
-#[tokio::test(flavor = "multi_thread")]
+#[test_log::test(tokio::test(flavor = "multi_thread"))]
```

Comment:
Change makes sense! Ideally, for next time, it would be a separate PR, mixing multiple unrelated changes outs more mental load on reviewers than two separate PRs.